tidb-server/main.go

```diff
@@ -360,6 +360,8 @@ func setGlobalVars() {
 	if cfg.TiKVClient.GrpcConnectionCount > 0 {
 		tikv.MaxConnectionCount = cfg.TiKVClient.GrpcConnectionCount
 	}
+
+	tikv.CommitMaxBackoff = int(parseLease(cfg.TiKVClient.CommitTimeout).Seconds() * 1000)
```

Comment:
`parseLease` is not accurate any more, `parseDuration`  looks better. The rest LGTM.

Reply:
ok